.github/workflows/ci.yml

```diff
@@ -3,41 +3,42 @@ name: Compile and run tests
 on: [push, pull_request]
 
 jobs:
-  lint:
-    container: "registry.fedoraproject.org/fedora:latest"
+  lint-and-build:
```

Comment:
Note that changing the lint names breaks the required lints. It would probably make sense to open a separate PR to fix CI like #43 was attempting.

Note that I don't have access to change the required lints. I think @Conan-Kudo can do that, so it's a hard blocker for any PR if you'd like for me to review it.

Reply:
Not sure I follow, please let me know how to proceed 👍

Reply:
The `main` branch is protected, and has requirements set for the lints whose names you are attempting to change.

I don't have access to make changes to that, and unless someone with access does change them, it can't be merged.

Reply:
This is all done, this PR needs to be rebased.

Reply:
@Conan-Kudo  rebased